lib/patches/config_patch.rb

```diff
@@ -11,3 +11,4 @@ def reload!
 end
 
 Config::Options.prepend(ConfigDecryptPasswords)
+Config::Options.include(MoreCoreExtensions::Shared::Nested)
```

Comment:
@abellotti I don't *think* so....I think all these methods (including fetch_path) use `[]` to do the lifting, which `Config::Options` responds to